sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SignResult.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    /// <summary>
+    /// Represents information about a sign operation
+    /// </summary>
+    public struct SignResult : IJsonDeserializable
```

Comment:
Should we hide gethashcode and equals from these structs? using EditorBrowsable? Should these struct implement IEquatable?

Reply:
Again as above, I reverted to making these classes